src/components/DocumentationBrowser/DocumentationBrowser.tsx

```diff
@@ -150,7 +184,13 @@ export function DocumentationBrowser() {
         <CardBody>
           <Accordion allowMultiple>
             {actionProviderMenu.main.map((ap) => (
-              <ActionProviderItem ap={ap} key={ap.url} />
+              <ActionProviderItem
+                ap={ap}
+                key={ap.url}
+                addToState={(apState) => {
+                  addToState(ap?.definition?.title || ap.url, apState);
+                }}
```

Comment:
We can take the stance that if there is no `definition` (at the moment this means the AP isn't sending proper CORs headers), there is no `Add to Flow` button, which I think eliminates this conditional.